build.xml

```diff
@@ -55,6 +70,7 @@
     <javac debug="on" srcdir="browse-handler/java" destdir="${builddir}/browse-handler">
       <classpath refid="classpath"/>
       <compilerarg value="-Xlint"/>
+      <compilerarg value="-Xdiags:verbose"/>
```

Comment:
Is this verbose flag intentionally part of the commit, or a relic of past debugging? I don't mind keeping it, just want to be sure it's meant to be here.

Reply:
A relic of the past, but very useful for certain library or reflective programming failures.

Reply:
Will it impact the final generated .jar file in any way? If it makes the build process more verbose, I have no objections at all. If it affects the behavior of the finished product, it might be best to control it with a switch in case it introduces any security vulnerabilities from deploying a debug-mode object into production. (I'm guessing that this is not the case -- but just double-checking out of healthy paranoia).

Reply:
It is just verbose. As some point some of the complier messages were simplified. When you run into that, you get the message:
>Some messages have been simplified; recompile with -Xdiags:Verbose to get full output

which is the clue that some messages have been suppressed. I've commented this out for now, but if there's ever a need to turn it back on, I would just leave it on. We really don't want warnings in the compile messages anyhow, and best to be explicit when they show up.